static/src/javascripts/projects/common/modules/commercial/adverts.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'bean',
+    'qwery',
+    'fastdom'
+], function (bean, qwery, fastdom) {
+    var shouldWePolyfill = !('open' in document.createElement('details'));
+
+    return {
+        init: init
+    };
+
+    function init() {
+        if (shouldWePolyfill) {
+            bean.on(document, 'click', 'summary', onClick);
+            bean.on(document, 'keypress', 'summary', onKeyPress(onClick));
+        }
+
+        var showMores = qwery('.adverts__more > summary');
+        bean.on(document, 'click', showMores, onOpenClick);
+        bean.on(document, 'click', showMores, onKeyPress(onOpenClick));
+    }
+
+    function onClick(event) {
+        var details = event.currentTarget.parentNode;
+        if (details.hasAttribute('open')) {
+            fastdom.write(function () {
+                details.removeAttribute('open');
+            });
+        } else {
+            fastdom.write(function () {
+                details.setAttribute('open', 'open');
+            });
+        }
+    }
+
+    function onKeyPress(handler) {
+        return function (event) {
+            if (event.keyCode === 0x20 || event.keyCode === 0x0D) {
+                handler(event);
+            }
+        };
+    }
+
+    function onOpenClick(event) {
+        var summary = event.currentTarget;
+        var details = summary.parentNode;
+        var label = summary.querySelector('.button__label');
```

Comment:
(1) you're right, I'm changing it.
(2) if it was me, yes, not only because it is the right element, but also because this 'More/Less' behaviour should be removed... it is bad UX.